tests/php/includes/class-wpjm-base-test.php

```diff
@@ -166,19 +166,21 @@ protected function assertNotPostStatus( $expected_post_type, $post ) {
 		$this->assertNotEquals( $expected_post_type, $post->post_status );
 	}
 
-	protected function get_user_by_role( $role ) {
+	protected function get_user_by_role( $role, $variant = '' ) {
```

Comment:
do we need the `$variant` part of it? could we get away with e.g. 

```
return $this->login_as( $this->get_user_by_role( 'employer_b' ) );
```

Reply:
We could, but we'd need to parse the string to determine the actual role to set with `set_role`. This seems a bit cleaner.